lib/screens/signup/signup.dart

```diff
@@ -204,7 +207,13 @@ class SignupScreen extends StatelessWidget {
                               ),
                             ),
                             onTap: () {
-                              // TODO
+                              Navigator.pushReplacement(
+                                  context,
+                                  MaterialPageRoute(
+                                      builder: (_) => BlocProvider(
+                                            create: (_) => LoginCubit(FakeLoginRepository()),
```

Comment:
I believe APILoginRepository is ready. Use that instead of Fake.

Reply:
I replaced it for FakeSignUpRepository as well.